### ISSUE TITLE
Examples shown in arquillian containers not working

### ISSUE THREAD

**Issue by phillipross** [payara/Payara-Community-Documentation#45]
I discovered problems in the payara arquillian container documentation after struggling to resolve this issue:
https://github.com/payara/ecosystem-arquillian-connectors/issues/51

I previously filed issues ([here](https://github.com/payara/Payara-Server-Documentation/issues/709) and [here](https://github.com/payara/Payara-Community-Documentation/issues/12)) for updating the documentation which have since been closed as the documentation has moved and has seen updates, but the issue remains that the documentation has non-working examples.

To summarize the details and conclusions from the previous issues, the examples in the arquillian container documentation show container properties defined as system properties in the surefire/failsafe plugin, but defining here does not work.  The properties need to be defined in arquillian.xml to work properly.  I believe the examples in the documentation should be changed to state this.
